lib/internal/Magento/Framework/Validator/Test/Unit/Constraint/Option/CallbackTest.php

```diff
@@ -28,41 +27,46 @@ class CallbackTest extends \PHPUnit\Framework\TestCase
      */
     public function testGetValue($callback, $expectedResult, $arguments = null, $createInstance = false)
     {
-        $option = new \Magento\Framework\Validator\Constraint\Option\Callback($callback, $arguments, $createInstance);
-        $this->assertEquals($expectedResult, $option->getValue());
+        $option = new Callback($callback, $arguments, $createInstance);
+        self::assertEquals($expectedResult, $option->getValue());
     }
 
     /**
      * Data provider for testGetValue
      */
     public function getConfigDataProvider()
     {
-        $functionName = create_function('', 'return "Value from function";');
         $closure = function () {
             return 'Value from closure';
         };
 
-        $mock = $this->getMockBuilder('Foo')->setMethods(['getValue'])->getMock();
-        $mock->expects(
-            $this->once()
-        )->method(
-            'getValue'
-        )->with(
-            'arg1',
-            'arg2'
-        )->will(
-            $this->returnValue('Value from mock')
-        );
+        $mock = $this->getMockBuilder('Foo')
+            ->setMethods(['getValue'])
+            ->getMock();
+        $mock->method('getValue')
```

Comment:
Looks like section `->expects($this->once)` is missing here

Reply:
It doesn't matter how many times `getValue` will be called, that's why this method is not needed.

Reply:
In this case would be use `->expects($this->any())`

Reply:
For this test case, in general, it doesn't matter how many times the method will be called, so `any()` is redundant. The `expects` may sense to use when we need to know if the method is called or not (corner cases, like business logic conditionals) or how many times it was called - for performance reasons.
I don't see, how `any()` makes test a cleaner.

Reply:
Ok. Good point. Let's leave it as is